src/app/views/templates/MrfWorkflowCompletionEmail.tsx

```diff
@@ -1,72 +1,94 @@
 import {
   Body,
-  Column,
   Container,
   Head,
   Heading,
   Hr,
   Html,
   Img,
-  Row,
+  Preview,
+  Section,
   Text,
 } from '@react-email/components'
 import { FORMSG_LOGO_URL } from '../../constants/formsg-logo'
 
 import {
-  headingStyle,
-  innerContainerStyle,
-  outerContainerStyle,
-  textStyle,
+  bodyTextStyle,
+  containerStyle,
+  headingTextStyle,
+  mainStyle,
+  outcomeTextStyle,
+  sectionStyle,
+  titleTextStyle,
 } from './styles'
 
+export enum WorkflowOutcome {
+  APPROVED = 'Approved', 
+  NOT_APPROVED = 'Not approved' 
+}  
+
+export type QuestionAnswer = {
+  question: string, 
+  answer: string 
+}
+
 export type WorkflowEmailData = {
   formTitle: string
   responseId: string
+  formQuestionAnswers: QuestionAnswer[]
+  outcome?: WorkflowOutcome | undefined 
 }
 
 export const MrfWorkflowCompletionEmail = ({
   // Defaults are provided only for testing purposes in react-email-preview.
   formTitle = 'Test form title',
-  responseId = '64303c45828035f732088a41'
+  responseId = '64303c45828035f732088a41', 
+  formQuestionAnswers = [], 
+  outcome
 }: WorkflowEmailData): JSX.Element => {
+  const previewText = outcome
+    ?  `The outcome for ${formTitle}.` 
+    : `${formTitle} has been completed by all respondents.` 
+
   return (
     <Html>
-      <Head />
-      <Body style={outerContainerStyle}>
-        <Container style={innerContainerStyle}>
-          <Row>
-            <Column>
-              <Img src={FORMSG_LOGO_URL} alt="FormSG" />
-            </Column>
-          </Row>
-          <Row style={{ paddingTop: '32px' }}>
-            <Column>
-              <Heading style={headingStyle}>
-                {formTitle} has been completed by all respondents.
+      <Head /> 
+      <Preview>{previewText}</Preview>
+      <Body style={mainStyle}>
+          <Container style={containerStyle}>
+            <Section style={sectionStyle}>
+              <Img style={{height: '1.5rem', marginBottom: '2.5rem'}} src={FORMSG_LOGO_URL} alt="FormSG" />
+              <Heading style={headingTextStyle}>
+                {
+                  outcome 
+                    ? `The outcome for ${formTitle}` 
+                    : `${formTitle} has been completed by all respondents.` 
+                }
+              </Heading>
+              {
+                outcome ? <>
+                  <Text style={{...outcomeTextStyle, marginTop: '2.5rem', marginBottom: '0.25rem'}}>Outcome</Text>
```

Comment:
Might be able to utilise nested `<Text />` here as child `<Text />` should inherit parent's styles. 
```
<Text style={outcomeTextStyle}>
  <Text style={{marginTop: '2.5rem', marginBottom: '0.25rem'}}>Outcome</Text>
  <Text style={{fontWeight: 400, marginTop: '0.25rem'}}>{outcome}</Text>
</Text>
```

Reply:
Doing this in react email causes the style to not apply, need to apply each style on each "component" individually.

Reply:
though i understand your intention of reducing code duplication and using common variables to make future updates simpler, i have reduced duplications as much as possible.